### PR TITLE
Update #62403 itest.5ch.net

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/pull/62403
Removed cosmetic filters added to EL as generic ones and added an unblocked adserver.

URL: `http://itest.5ch.net/test/read.cgi/android/1604314941/`

<details>
<summary>Screenshot</summary>

itest

</details>

### CODE REVIEW

File: JapaneseFilter/sections/adservers.txt [hunk 1]
```diff
@@ -63,6 +63,7 @@
 ! Eimg.jp
 ||adingo.jp.eimg.jp^
 !
+||amoad.com^$third-party
```
Comment (zloyden) [inline JapaneseFilter/sections/adservers.txt:66]: We have a similar rule in `AdGuard Mobile` filter. I will remove it after closing this pull request.

**Review by zloyden — APPROVED** [review]

File: JapaneseFilter/sections/adservers.txt (hunk 1 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/adservers.txt:66]: Ah, sorry I forgot to enable Mobile filter on test.

Comment (zloyden) [reply JapaneseFilter/sections/adservers.txt:66]: Yeah, it happens. Have you noticed the use of this server in countries other than Japan?

Comment (Yuki2718) [reply JapaneseFilter/sections/adservers.txt:66]: I think only Japanese sites. but it's in my personal list for desktop, suggesting it's used on some desktop site. I can quickly search for.

Comment (Yuki2718) [reply JapaneseFilter/sections/adservers.txt:66]: Here two examples:
```
https://www.carsensor.net/
https://retty.me/
```

Comment (zloyden) [reply JapaneseFilter/sections/adservers.txt:66]: Ok, thanks. Then it will stay for Japan.

**Comment by Yuki2718** [comment]
Some filters I removed are not in the optimized version of Base. Pinged @ameshkov .

**Pull request merged**